### ISSUE TITLE
terraform apply fails on hosts without bash

### ISSUE THREAD

**Issue by andytheapedemontague** [terraform-google-modules/terraform-google-cloud-operations#41]
### TL;DR

Module fails on hosts without bash installed; the module appears to depend on running on a host with bash installed (sh is not sufficient).

### Expected behavior

Module should execute successfully on hosts without Bash e.g. the Cloud Build Terraform builder.

### Observed behavior

Executing a Terraform script with this module on a host that doesn't have Bash installed (e.g. the Cloud Build Terraform builder, which is based on Alpine which AFAIU does not come with Bash) fails. The error is something like the below.

I believe the root cause would be the hash-bang in scripts/check_components.sh is `#!/usr/bin/env bash`

```
Step #3 - "tf apply": module.agent_policy.module.gcloud-upsert.null_resource.additional_components[0]: Provisioning with 'local-exec'...
Step #3 - "tf apply": module.agent_policy.module.gcloud-upsert.null_resource.additional_components[0] (local-exec): Executing: ["/bin/sh" "-c" ".terraform/modules/agent_policy.gcloud-upsert/scripts/check_components.sh gcloud alpha"]
Step #3 - "tf apply": module.agent_policy.module.gcloud-destroy.null_resource.additional_components[0]: Provisioning with 'local-exec'...
Step #3 - "tf apply": module.agent_policy.module.gcloud-destroy.null_resource.additional_components[0] (local-exec): Executing: ["/bin/sh" "-c" ".terraform/modules/agent_policy.gcloud-destroy/scripts/check_components.sh gcloud alpha"]
Step #3 - "tf apply": module.agent_policy.module.gcloud-destroy.null_resource.additional_components[0] (local-exec): env: can't execute 'bash': No such file or directory
Step #3 - "tf apply": module.agent_policy.module.gcloud-upsert.null_resource.additional_components[0] (local-exec): env: can't execute 'bash': No such file or directory
Step #3 - "tf apply": ╷
Step #3 - "tf apply": │ Error: local-exec provisioner error
Step #3 - "tf apply": │ 
Step #3 - "tf apply": │   with module.agent_policy.module.gcloud-destroy.null_resource.additional_components[0],
Step #3 - "tf apply": │   on .terraform/modules/agent_policy.gcloud-destroy/main.tf line 174, in resource "null_resource" "additional_components":
Step #3 - "tf apply": │  174:   provisioner "local-exec" {
Step #3 - "tf apply": │ 
Step #3 - "tf apply": │ Error running command
Step #3 - "tf apply": │ '.terraform/modules/agent_policy.gcloud-destroy/scripts/check_components.sh
Step #3 - "tf apply": │ gcloud alpha': exit status 127. Output: env: can't execute 'bash': No such
Step #3 - "tf apply": │ file or directory
Step #3 - "tf apply": │ 
Step #3 - "tf apply": ╵
Step #3 - "tf apply": ╷
Step #3 - "tf apply": │ Error: local-exec provisioner error
Step #3 - "tf apply": │ 
Step #3 - "tf apply": │   with module.agent_policy.module.gcloud-upsert.null_resource.additional_components[0],
Step #3 - "tf apply": │   on .terraform/modules/agent_policy.gcloud-upsert/main.tf line 174, in resource "null_resource" "additional_components":
Step #3 - "tf apply": │  174:   provisioner "local-exec" {
Step #3 - "tf apply": │ 
Step #3 - "tf apply": │ Error running command
Step #3 - "tf apply": │ '.terraform/modules/agent_policy.gcloud-upsert/scripts/check_components.sh
Step #3 - "tf apply": │ gcloud alpha': exit status 127. Output: env: can't execute 'bash': No such
Step #3 - "tf apply": │ file or directory
Step #3 - "tf apply": │ 
Step #3 - "tf apply": ╵
```

### Terraform Configuration

```hcl
module "agent_policy" {
  source     = "terraform-google-modules/cloud-operations/google//modules/agent-policy"
  version    = "~> 0.1.0"

  project_id = local.project_name
  policy_id  = "ops-agent-policy"
  agent_rules = [
    {
      type               = "ops-agent"
      version            = "current-major"
      package_state      = "installed"
      enable_autoupgrade = true
    },
  ]
  os_types = [
    {
      short_name = "ubuntu"
    },
  ]
}
```

### Terraform Version

```sh
Terraform v1.2.3
on linux_amd64
+ provider registry.terraform.io/hashicorp/external v2.2.2
+ provider registry.terraform.io/hashicorp/google v4.12.0
+ provider registry.terraform.io/hashicorp/null v3.1.1
+ provider registry.terraform.io/hashicorp/random v3.3.2
+ provider registry.terraform.io/hashicorp/time v0.7.2
```
